docs/deployment/upgrade.md

```diff
@@ -8,53 +8,60 @@ hide_title: true
 
 After the initial deployment, any future updates to the Cumulus deployment from configuration files, Terraform files (`*.tf`), or modules from a new version of Cumulus can be deployed and will update the appropriate portions of the stack as needed.
 
-## Cumulus versioning
+## Cumulus Versioning
 
-Cumulus uses a global versioning approach, meaning version numbers are consistent across all packages and tasks, and semantic versioning to track major, minor, and patch version (i.e. 1.0.0). We use Lerna to manage versioning.
+Cumulus uses a global versioning approach, meaning version numbers are consistent across all packages and tasks, and semantic versioning to track major, minor, and patch version (e.g., 1.0.0). We use Lerna to manage versioning.
 
-To view the released module artifacts for each Cumulus core version, see the [Cumulus releases page](https://github.com/nasa/cumulus/releases).
+To view the released module artifacts for each Cumulus core version, see the [Cumulus Releases] page.
 
-**IMPORTANT:** Cumulus releases may introduce breaking changes between minor versions (e.g. 1.13.x -> 1.14.0), so make sure to consult the release notes for migration steps.
+**IMPORTANT:** Cumulus releases may introduce breaking changes between minor versions (e.g., 1.13.x -> 1.14.0), so it is critical that you consult the release notes for migration steps.
```

Comment:
I think we should be more explicit that our minor versions *are* breaking change versions.   We shouldn't be upping the minor version unless there is a breaking change.     Something along the lines of, but not precisely:  

```suggestion
**IMPORTANT:** By convention, Cumulus minor version releases introduce breaking changes (e.g., 1.13.x -> 1.14.0), so it is critical that you consult the release notes for migration steps.
```

It might be a good idea to also explicitly note that our x.y.z format doesn't follow SemVer, but that's more of a personal preference than a generally applicable PR comment.

Reply:
I agree with Jonathan, but I think that should go up to the first paragraph that says we use semantic versioning.